### ISSUE TITLE
multi-column is ignored on IE11

### ISSUE THREAD

**Issue by MurakamiShinyu** [vivliostyle/vivliostyle.js#210]
multi-column properties (specified outside page partitions) are ignored on IE10/IE11. (it seems column-\* properties are not passed to the browser. Note that IE10/IE11 supports css3 multicol)

[Test sample](https://www.dropbox.com/s/nyjvttm861if28i/bugtest3.html?dl=0)

**Comment by MurakamiShinyu** [comment]
The problem found in src/adapt/base.js:

```
adapt.base.makePropNameMap = function(list) {
    var map = {};
    var probe = document.createElement("span");
    var style = probe.style;
    var lastPrefix = null;
    try {
        probe.style.setProperty("-ms-transform-origin", "0% 0%");
        if (probe.style.getPropertyValue("-ms-transform-origin") == "0% 0%") {
            for (var i = 0; i < list.length; i++) {
                map[list[i]] = "-ms-" + list[i];
            }
            return map;
        }
```

and

```
adapt.base.propNameMap = adapt.base.makePropNameMap([
    "transform", "transform-origin", "hyphens", "writing-mode",
    "text-orientation", "box-decoration-break",
    "column-count", "column-width", "column-rule-color",
    "column-rule-style", "column-rule-width", ...
```

This code adds "-ms-" prefix to all "column-count", "column-width", etc. but IE don't support "-ms-" prefixed column-\* properties. IE supports just "column-*" properties.

**Comment by MurakamiShinyu** [comment]
MS Edge doesn't have this problem because Edge no longer supports "-ms-transform-origin", and therefore has another problem, hyphenation is ignored on Edge (Edge supports "-ms-hyphens" and not "hyphens").

A workaround would be:

```
        probe.style.setProperty("-ms-hyphens", "auto");
        if (probe.style.getPropertyValue("-ms-hyphens") == "auto") {
            for (var i = 0; i < list.length; i++) {
                var msJsName = adapt.base.cssToJSProp("ms-" + list[i]);
                if (style[msJsName] != null)
                    map[list[i]] = "-ms-" + list[i];
                else
                    map[list[i]] = list[i];
            }
            return map;
        }
```

**Comment by kwkbtr** [comment]
Fixed by #215.
